notebooks/StatiGraphics.jl

```diff
@@ -0,0 +1,164 @@
+### A Pluto.jl notebook ###
+# v0.12.20
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 01c88650-1fa3-11eb-1156-bb632230292b
+using Random; Random.seed!(2019); # make sure this tutorial is reproducible
+
+# ╔═╡ 221618e0-6e73-11eb-047e-3b329eefa7d3
+begin
+	using GeoStats
+	using StratiGraphics
+	using SpectralGaussianSimulation
```

Comment:
This is not needed anymore, the FFTGS solver is loaded by default in the latest releases.

Reply:
Removing `using SpectralGaussianSimulation` is giving the following error:
```
UndefVarError: FFTGS not defined
```

Reply:
Oh I think it is because we need to update the environment to use the latest GeoStats.jl then. Could you please try to `]up` the environment in the notebook folder? The latest version of the project is v0.21 if I remember correctly.

After the update, please open all notebooks to make sure that everything is working as expected still.

Reply:
Also, I don't understand how you are able to use SpectralGaussianSimulation without having it in the Project.toml in this PR. Please make sure you are running the notebook from the notebooks folder. There is a specific environment set there.

Reply:
The problem is that Pluto isn't using the current activated environment, instead, it is using the standard environment. Looks like it is a known issue and I'm not able to load Pluto in the notebooks environment. Please could you try running the notebook on your side?

Reply:
That is why I asked you to copy/paste the header cells of other notebooks in this repository. Those cells will take care of using the environment inside the folder. Can you please update the notebook here to include the header cells present in all notebooks?

After that you will be able to update the environment if it is still needed.

Reply:
Try to:

1. Include the header cells
2. Run the notebook without SpectralGaussianSimulation.jl
3. If the error persists, update the environment of the folder notebooks:
```julia
] activate notebooks
] up
```

Reply:
Yes, that worked